File: src/main/scala/com/shmigel/dotamatchbot/util/JsonParser.java
package com.shmigel.dotamatchbot.util;

import com.google.gson.Gson;
import com.shmigel.dotamatchbot.service.telegram.TelegramResponse;

public class JsonParser {

    private static Gson parser = new Gson();

    public static TelegramResponse parse(String json) {
        return parser.fromJson(json, TelegramResponse.class);
    }

}

File: src/main/scala/com/shmigel/dotamatchbot/util/JsonParser.scala
package com.shmigel.dotamatchbot.util

import com.google.gson.Gson
import com.shmigel.dotamatchbot.service.telegram.TelegramResponse

object JsonParser {

  private val parser = new Gson()

  def parse(json: String): TelegramResponse = {
    parser.fromJson(json, classOf[TelegramResponse])
  }

}
